app/assets/javascripts/calendar.es6

```diff
@@ -1,9 +1,12 @@
-/* global moment */
+/* global moment, cookie */
 
 'use strict';
 
 class Calendar {
   constructor(el, config = {}) {
+    this.cookieName = this.constructor.name;
```

Comment:
Worth having this as part of the config? In the case of multiple calendars on a page etc.